### PR TITLE
Update to stable 2.11.4

### PR DESCRIPTION
**Pull request by webwurst**
This PR:

- adds/changes/removes etc

### Checklist

- [ ] Update changelog in CHANGELOG.md
- [ ] Run manual tests

### CODE REVIEW

File: helm/linkerd2-multicluster-app/values.yaml [hunk 1]
```diff
@@ -54,6 +54,3 @@ enablePSP: true
 # across hosts and zones for High Availability.
 # Enable this only when you have multiple replicas of components.
 enablePodAntiAffinity: true
-
-# -- User ID for the gateway
-controllerUID: 2103
```
Comment (pipo02mix) [inline helm/linkerd2-multicluster-app/values.yaml:59]: does ti work without controllerUID?

Comment (webwurst) [reply helm/linkerd2-multicluster-app/values.yaml:59]: It is not part of upstream [2.11.4](https://github.com/linkerd/linkerd2/blob/stable-2.11.4/multicluster/charts/linkerd-multicluster/values.yaml) and they changed it to a different key for [2.12.0](https://github.com/linkerd/linkerd2/blob/stable-2.12.0/multicluster/charts/linkerd-multicluster/values.yaml#L30)..

**Review by pipo02mix — APPROVED** [review]
LGTM if tested

**Comment by mcharriere** [comment]
Minimal upgrade done in PR #32. I'll close this one for now. the charts will require a bit of rework in prep for 2.12

**Pull request closed without merging**